cmd/tk/flags.go

```diff
@@ -13,11 +13,13 @@ import (
 )
 
 type workflowFlagVars struct {
+	name    string
 	targets []string
 }
 
 func workflowFlags(fs *pflag.FlagSet) *workflowFlagVars {
 	v := workflowFlagVars{}
+	fs.StringVar(&v.name, "name", "", "selects an environment from inline environments")
```

Comment:
From a usage perspective, I think it would be slightly nicer to have the environments name as a second argument, compared to a flag:

```
tk apply ./environments/single
tk apply ./environments/multi my-env
```

For `tk export` that wouldn't work of course, the flag the best solution here

Reply:
I would choose for consistency here.